query-engine/connector-test-kit/src/test/scala/writes/relations/OptionalRelationSpec.scala

```diff
@@ -61,8 +61,8 @@ class OptionalRelationSpec extends FlatSpec with Matchers with ApiSpecBase {
         | }
       """,
       project,
-      errorCode = 2012,
-      errorContains = "Missing a required value at `Mutation.updateList.data.ListUpdateInput.todo"
+      errorCode = 2009,
+      errorContains = "`Mutation.updateList.data.ListUpdateInput.todo`: A value is required but not set."
```

Comment:
The old error code seems to be the more correct one to me. Was this conscious or is the old one not possible anymore? One could even consider this to be a breaking change.
Maybe we still do it that way. We should just get everyone onboard with that change.

Reply:
Not possible to represent the other one anymore, unions change error handling in many ways.

Reply:
To add to this: These types of changes are not new. This has been going on since union types.